admin/metrics/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class OSFStatistic(models.Model):
```

Comment:
Let's find a better name for this class. First, this is the `osf.io` repo, we can assume that any class we're defining is related to OSF. Indeed, it looks like prefixing models and views with `OSF` is idiosyncractic here:

```
$ grep "class OSF" `find {admin,website} -name models.py -or -name views.py`
admin/metrics/models.py:class OSFStatistic(models.Model):
admin/metrics/views.py:class OSFStatisticsListView(ListView):
```

Secondly, the singular "statistic" is misleading, since this is really a collection of statistics tied to a particular date. How about `StatisticsForDate` instead?